metabase/metabase
# Do not put js code into page title

Closes https://github.com/metabase/metabase/issues/40051

### Description

After latest changes we converted function into function creator and didn't update all the places of usage.
From now we show a message instead of js code in the page title

### How to verify

- create a native question
- select postgres db and put `select pg_sleep(60)`;
- click "run" and check what's written in the tab title, there shouldn't be something liek `e => e..` there, but a message `Doing science ...`


### Checklist
